src/styles/index.scss

```diff
@@ -1,3 +1,35 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+@import './blocks/header';
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+  box-sizing: border-box;
```

Comment:
Consider using a variable for `box-sizing` to maintain consistency and ease of maintenance. You can define a variable like `$box-sizing: border-box;` in your `variables.scss` and then use it here as `box-sizing: $box-sizing;`.